images/tidb-backup-manager/Dockerfile

```diff
@@ -37,8 +37,8 @@ RUN \
   wget -nv https://download.pingcap.org/tidb-toolkit-${TOOLKIT_V40}-linux-amd64.tar.gz \
   && tar -xzf tidb-toolkit-${TOOLKIT_V40}-linux-amd64.tar.gz \
   && mv tidb-toolkit-${TOOLKIT_V40}-linux-amd64/bin/br /usr/local/bin/br40 \
-  && mv tidb-toolkit-${TOOLKIT_V40}-linux-amd64/bin/mydumper /mydumper \
-  && chmod 755 /usr/local/bin/br40 /mydumper \
+  && mv tidb-toolkit-${TOOLKIT_V40}-linux-amd64/bin/dumpling /dumpling \
```

Comment:
Can we combine the L19-L26 block with this block?

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/2870/commits/aa532c90288690796c5079d59a7cc3f817705b4f